lib/services/github.ts

```diff
@@ -333,6 +334,45 @@ export class GithubService extends WorkerClient<string> implements ServiceListen
         };
     }
 
+    public getConfigurationFile(details: GithubConfigLocation): Promise<string | void> {
```

Comment:
Any reason this is passed the whole `details` object, and not just `details.location`?